lib/MakeExt2Image.pm

```diff
@@ -1,73 +1,51 @@
 #! /usr/bin/perl -w
 
-# Create an empty ext2 fs.
-#
-# Usage:
-#
-#   use MakeExt2Image;
-#
-#   exported functions:
-#     MakeExt2Image(file_name, size_in_kbyte, inodes);
+=head1 NAME
 
+MakeExt2Image - create ext2 file system.
 
-=head1 MakeExt2Image
+=head1 SYNOPSIS
 
-C<MakeExt2Image.pm> is a perl module that can be used to create Ext2 file
-systems. It exports the following symbols:
+  use MakeExt2Image;
 
-=over
-
-=item *
+  # create a 5000 kbyte file system with 300 inodes.
```

Comment:
... shouldn't we be explicit about KiB (1024 B) vs kB (1000B)?
At least in the newly added docs.